portfolio/src/main/webapp/script.js

```diff
@@ -15,10 +15,41 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  var quote ='I am trilingual';
 
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = quote;
+async function ShowFact(contentnum){
+    const responseFromServer = await fetch('/hello');
+    const textFromResponse = await responseFromServer.json();
+    var facts = [textFromResponse.fact1, textFromResponse.fact2, textFromResponse.fact3];
+    var factclick = [0,0,0];
+    var container = "hello-container1";
+    var contents = facts[0];
+        
+    if (contentnum == 1){
```

Comment:
Rather than branching like this, you can simply do:

```javascript
contents = facts[contentnum-1];
container = `hello-container${contentnum}`;
factclick[contentnum-1]++;
```
Note the use of a template string for `container`